src/sso/sso.ts

```diff
@@ -45,4 +46,9 @@ export class SSO {
 
     return data.profile as Profile;
   }
+
+  async promoteDraftConnection({ token }: PromoteDraftConnectionOptions) {
+    const endpoint = '/draft_connections/' + token + '/activate';
```

Comment:
Can we use template strings instead? and call it `Id` instead of `token` (we usually refer to token internally for authentication)?

```
const endpoint = `/draft_connections/${Id}/activate`;
```